src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -431,11 +434,15 @@ class ConfigureController {
         document.setMargins(20.0f, 20.0f, 30.0f, 30.0f)
         document.newPage()
 
-        val table = PdfPTable(export.getHeader().size)
-        table.setWidths(export.getWidths())
+        val table = PdfPTable(export.header.size)
+        table.setWidths(export.widths)
         table.widthPercentage = 100f
         addTableHeader(table, export)
-        addRows(table, export, opening)
+        val orderRows = addOrderRows(table, export, opening)
+        addEmptyRows(table, export, emptyRows)
+        if ((emptyRows + orderRows) == 0) { // <-- In case of empty document
```

Comment:
If you want to clarify a condition you can always extract it to a method. 
Tip: Select the condition, `Ctrl+Alt+M`, enter something like `isDocumentEmpty`, hit enter